descriptors/ModuleDescriptor-template.json

```diff
@@ -148,6 +156,16 @@
       "permissionName": "lists.configuration.get",
       "displayName": "Lists: Get configuration of list application",
       "description": "Get configuration of lists application"
+    },
+    {
+      "permissionName": "lists.item.versions.collection.get",
+      "displayName": "Lists: Get previous versions of a list",
+      "description": "Get the previous versions of a list"
+    },
+    {
+      "permissionName": "lists.item.contents.get",
+      "displayName": "Lists: Get a previous version of a list",
+      "description": "Get a previous version of a list"
     }
```

Comment:
This looks like a duplicate of an existing permission (but with a different display name/description)